index.js

```diff
@@ -1,85 +1,108 @@
+
+    function untrap() {
+        if (trappedEl) {
+            topTrap = body.removeChild(topTrap);
+            outerTrapBefore = trappedEl.parentNode.removeChild(outerTrapBefore);
+            innerTrapBefore = trappedEl.removeChild(innerTrapBefore);
+            innerTrapAfter = trappedEl.removeChild(innerTrapAfter);
+            outerTrapAfter = trappedEl.parentNode.removeChild(outerTrapAfter);
+            botTrap = body.removeChild(botTrap);
+
+            trappedEl.classList.remove('keyboard-trap--active');
+
+            // let observers know the keyboard is now trapped
+            var event = document.createEvent('Event');
+            event.initEvent('keyboardUntrap', false, true);
+            trappedEl.dispatchEvent(event);
+
+            trappedEl = null;
+        }
+        return trappedEl;
     }
-    return trappedEl;
-}
 
-function trap(el) {
-    untrap();
+    function trap(el) {
+        untrap();
 
-    trappedEl = el;
+        trappedEl = el;
 
-    var focusableElements = trappedEl.querySelectorAll(util.focusableElementsList);
-    firstFocusableElement = focusableElements[0];
-    lastFocusableElement = focusableElements[focusableElements.length - 1];
+        var focusableElements = trappedEl.querySelectorAll(focusableElementsList);
+        firstFocusableElement = focusableElements[0];
+        lastFocusableElement = focusableElements[focusableElements.length - 1];
 
-    body.insertBefore(topTrap, body.childNodes[0]);
-    trappedEl.parentNode.insertBefore(outerTrapBefore, trappedEl);
-    trappedEl.insertBefore(innerTrapBefore, trappedEl.childNodes[0]);
-    trappedEl.appendChild(innerTrapAfter);
-    trappedEl.parentNode.insertBefore(outerTrapAfter, trappedEl.nextElementSibling);
-    body.appendChild(botTrap);
+        body.insertBefore(topTrap, body.childNodes[0]);
+        trappedEl.parentNode.insertBefore(outerTrapBefore, trappedEl);
+        trappedEl.insertBefore(innerTrapBefore, trappedEl.childNodes[0]);
+        trappedEl.appendChild(innerTrapAfter);
+        trappedEl.parentNode.insertBefore(outerTrapAfter, trappedEl.nextElementSibling);
+        body.appendChild(botTrap);
 
-    // let observers know the keyboard is now trapped
-    var event = document.createEvent('Event');
-    event.initEvent('keyboardTrap', false, true);
-    trappedEl.dispatchEvent(event);
+        // let observers know the keyboard is now trapped
+        var event = document.createEvent('Event');
+        event.initEvent('keyboardTrap', false, true);
+        trappedEl.dispatchEvent(event);
 
-    trappedEl.classList.add('keyboard-trap--active');
+        trappedEl.classList.add('keyboard-trap--active');
 
-    return trappedEl;
-}
+        return trappedEl;
+    }
 
-module.exports = {
-    trap: trap,
-    untrap: untrap
-};
+    return {
+        trap: trap,
+        untrap: untrap,
+        createTrapBoundary: createTrapBoundary,
+        focusableElementsList: focusableElementsList
```

Comment:
Same goes for here. Is there a reason for moving this out of the private util module? Also, I plan to integrate https://github.com/makeup-js/makeup-focusables

Reply:
since `utils` was "required" before, and it was fairly small, it seemed reasonable to move the util functions here and get rid of the need for `require` (which would limit this to bundlers only like lasso or webpack)

Reply:
We are okay limiting this to bundlers only. In fact, some makeup modules require other makeup modules, so it's a definite design decision. Keeping stuff in util also helps separate out concerns and identify common utility functions across the modules (like NodeListToArray).